3rdparty/python/requirements.txt

```diff
@@ -12,6 +12,7 @@ future==0.17.1
 futures==3.0.5 ; python_version<'3'
 Markdown==2.1.1
 mock==2.0.0
+more-itertools<6.0.0 ; python_version<'3'
```

Comment:
Consider adding `TODO(6282): once we can upgrade Pytest to 4.2.1+, we can remove this requirement`.